SimpleWAWS/Authentication/BaseAuthProvider.cs

```diff
@@ -68,7 +63,7 @@ public HttpCookie CreateSessionCookie(IPrincipal user)
         {
             var identity = user.Identity as TryWebsitesIdentity;
             SimpleTrace.Analytics.Information(AnalyticsEvents.UserLoggedIn, identity);
-            SimpleTrace.TraceInformation("{0}; {1}; {2}", AnalyticsEvents.OldUserLoggedIn, identity.Email, identity.Issuer);
+            SimpleTrace.TraceInformation("{0}; {1}; {2}", AnalyticsEvents.OldUserLoggedIn, identity?.Email, identity?.Issuer);
```

Comment:
Correct. I'll revert this. 
I dont recollect why this happened when debugging locally.